pyanaconda/modules/common/versions.py

```diff
@@ -0,0 +1,16 @@
+from functools import cmp_to_key
+
+import rpm
+
+
+__all__ = ['rpm_version_key']
+
+
+def rpm_version_cmp(v1, v2):
+    # TODO do we need to parse epoch? nothing in the tests indicates that
+    v1, _, r1 = v1.partition('-')
+    v2, _, r2 = v2.partition('-')
+    return rpm.labelCompare(('0', v1, r1), ('0', v2, r2))
+
+
+rpm_version_key = cmp_to_key(rpm_version_cmp)
```

Comment:
Please, move this code to `pyanaconda.core.payload`.

Reply:
Moved.